### DISCUSSION TITLE
translations/ API 제안

### DISCUSSION THREAD

**Discussion by jun-brro** [sunnybraille/sunnybraille-server#75] [Ideas]
**FileName 변수 추가**

<img width="428" alt="image" src="https://github.com/Sunflower-yonsei/sunflower-server/assets/115399447/34ab300b-e320-40a1-8cab-e67b3c8eca06">
<img width="385" alt="image" src="https://github.com/Sunflower-yonsei/sunflower-server/assets/115399447/b05ca583-5cf2-48ab-8a25-df81036aaaff">

저희는 ID로 파일 구분하는것이 명확하지만, 실제로 사용자 입장에서는 파일명을 보고 판단하고 싶어할 듯해서, 파일명도 추가적으로 보여주면 좋을 듯 싶습니다.

**Comment by gitchannn** [comment]
반영했습니다! 
https://github.com/Sunflower-yonsei/sunflower-server/pull/77
